### ISSUE TITLE
Training time for region based dataset,

### ISSUE THREAD

**Issue by mark-joe** [MIC-DKFZ/nnUNet#2844]
For the first time I did a region-based training, combining two specific classes from the 20 I have, and thus leaving 18 still as separate classes. nnU-Net delivered!  Dice score went up considerably for the combined class. Two questions:
1. Training time went up quite considerably: from 80 seconds per epoch to 220 seconds. This is not mentioned in the documentation of the Region-based training. Can this be OK?
2. To my surprise, the Dice scores of the other (single) classes also went up, albeit not much, about 0.02. Can this be explained? Maybe training is done somewhat differently, also leaving a positive influence on the single class 'regions'. 

Maybe someone can give a quick comment on this. Thank you!

**Comment by constantinulrich** [comment]
Hi :) 
1. Internally, the segmentation map is transformed to a one-hot-encoded multichannel map. depending on your system specification this can lead to an increased compute overhead for the CPU (especially for that many classes) -> check the GPU util to confirm that. Increasing the dataloader worker might help a little.
2.  When you do a region-based training, a sigmoid is applied instead of the softmax. So it is a binary decision for each class. This can also change the behavior of the other classes...
